services/object/src/backends/index.js

```diff
@@ -38,9 +35,17 @@ class Backends {
         rootUrl: cfg.taskcluster.rootUrl,
         config,
       });
+      
+      try{
+        await backend.setup();
+      }catch(err){
+        this.monitor.reportError(err.message);
+        continue;
+      }
+
       this._backends.set(backendId, backend);
```

Comment:
This looks good -- the effect is that the backend isn't in `_backends` if it didn't set up correctly.  That might leave some objects inaccessible, but that's the best we can do in the circumstances!